src/main/webapp/WEB-INF/views/profile.jsp

```diff
@@ -76,6 +86,7 @@
             </h1>
             <span class="text-sm">
                 Here you can delete your account. We're sad to see you leave, but you must have your reasons.
+                Don't worry, we'll sell your data to the NSA.
```

Comment:
Do we want to have this vibe ? If so, I suggest that we change more things. If we only have this sentence, I'd find it weird.